src/main/java/com/kodilla/calculator/Calculator.java

```diff
@@ -10,24 +10,35 @@ public double subtraction(double c, double d) {
         return c - d;
     }
 
-    public double multiplication (double e, double f) {
+    public double multiplication(double e, double f) {
         return e * f;
     }
 
-    public double division (double g, double h) {
+    public double division(double g, double h) {
         if (h != 0) {
             return g / h;
         } else {
             throw new IllegalArgumentException("Dont dive by zero");
         }
     }
 
+    public int fibonacci(int n) {
+        if (n == 1 || n == 0)
+            return n;
+
+        if (n < 0)
+            throw new IllegalArgumentException("Number must be 0 or higher.");
+
+        return fibonacci(n - 1) + fibonacci(n - 2);
```

Comment:
Podejście rekurencyjne jest jak najbardziej okej i w tym przypadku samo się narzuca :) ale ma jedną i to dość sporą wadę :) Czy probowałeś policzyć fibonacci(100000)?